.github/workflows/nightly.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
 
       - name: Test with Gradle
         uses: nick-invision/retry@v1
+        env:
+          S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ secrets.S3_BUILD_CACHE_ACCESS_KEY_ID }}
```

Comment:
Who's key is this? Hope it's CNCF's.

Reply:
right now - Trask :) We are still waiting on https://github.com/open-telemetry/community/issues/392

Reply:
yeah, we should get CNCF's soon, but wanted to unblock us in the short term